corehq/ex-submodules/casexml/apps/case/xform.py

```diff
@@ -23,6 +26,41 @@
 from dimagi.utils.logging import notify_exception
 
 
+class DirtinessFlag(object):
```

Comment:
A comment simon left on a PR reminded me of another really nice thing about `namedtuples`. When you print them (or put them in a logging message) the output is actually useful:

```python
>>> class Foo1(object):
...     def __init__(self, bar, baz):
...         self.bar = bar
...         self.baz = baz
...
>>> from collections import namedtuple
>>> Foo2 = namedtuple('Foo2', ['bar', 'baz'])
>>> Foo1(bar='green', baz='blue')
<__main__.Foo1 at 0x1082aec90>
>>> Foo2(bar='green', baz='blue')
Foo2(bar='green', baz='blue')
```